include/seqan3/contrib/charconv/charconv_pre.hpp

```diff
@@ -0,0 +1,57 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#pragma once
+
+#include <cassert>
+#include <system_error>
+#include <limits>
+
+#include <seqan3/core/platform.hpp>
+
+#define SEQAN3_CONTRIB_CHARCONV_LIBCPP_BEGIN_NAMESPACE_STD namespace seqan3::contrib::charconv {
+#define SEQAN3_CONTRIB_CHARCONV_LIBCPP_END_NAMESPACE_STD }
+// #define SEQAN3_CONTRIB_CHARCONV_LIBCPP_HAS_NO_PRAGMA_SYSTEM_HEADER 1 // needs to be set if compiling with MSVC
+// #define SEQAN3_CONTRIB_CHARCONV_LIBCPP_COMPILER_MSVC // needs to be set if compiling with MSVC
+#define SEQAN3_CONTRIB_CHARCONV_LIBCPP_STD_VER 17
+// #define SEQAN3_CONTRIB_CHARCONV_LIBCPP_PUSH_MACROS
+// #define SEQAN3_CONTRIB_CHARCONV_LIBCPP_POP_MACROS
+// #define SEQAN3_CONTRIB_CHARCONV_LIBCPP_AVAILABILITY_TO_CHARS
+#define SEQAN3_CONTRIB_CHARCONV_LIBCPP_FUNC_VIS
+#define SEQAN3_CONTRIB_CHARCONV_LIBCPP_ENUM_VIS
+#define SEQAN3_CONTRIB_CHARCONV_LIBCPP_TYPE_VIS
+// #define SEQAN3_CONTRIB_CHARCONV_LIBCPP_CONSTEXPR constexpr
+#define SEQAN3_CONTRIB_CHARCONV_LIBCPP_HIDDEN
+#define SEQAN3_CONTRIB_CHARCONV_LIBCPP_INLINE_VISIBILITY
+#define SEQAN3_CONTRIB_CHARCONV_LIBCPP_ASSERT(condition, message) assert(condition)
+
+// import things from std namespace which charconv uses
+namespace seqan3::contrib::charconv
+{
+using std::errc;
+using std::declval;
+using std::numeric_limits;
+using std::is_integral;
+using std::is_unsigned;
+using std::is_signed;
+// using std::make_unsigned;
```

Comment:
can you drop a comment why this is out commented?
Same for `using std::enable_if;`

Reply:
I can remove them :)

Reply:
I removed them :)